client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/settings.js

```diff
@@ -1,403 +0,0 @@
-/** @format */
```

Comment:
- Renamed this `client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/ups-settings-form.js` (since it's only used for UPS)
- Converted the component to use React Hooks.
- Rewrote the tests.